src/main/java/org/jitsi/jigasi/health/Health.java

```diff
@@ -110,6 +110,12 @@ public static void stop()
     public static void check()
         throws Exception
     {
+        if (!CallManager.isHealthy())
+        {
+            throw new Exception(
+                "CallManager blocked to answer or hangup calls.");
```

Comment:
I think just `CallManager is not healthy` should suffice as an explanation. `Health.java` shouldn't care about the specifics